tracer/src/Datadog.Trace/Debugger/LiveDebuggerFactory.cs

```diff
@@ -23,12 +26,18 @@ public static LiveDebugger Create(IDiscoveryService discoveryService, IRemoteCon
             return LiveDebugger.Create(settings, string.Empty, null, null, null, null, null, null);
         }
 
-        var apiFactory = DebuggerTransportStrategy.Get(settings.AgentUri);
-
         var snapshotStatusSink = SnapshotSink.Create(settings);
         var probeStatusSink = ProbeStatusSink.Create(settings, serviceName);
 
-        var batchApi = BatchUploadApiFactory.Create(settings, apiFactory, discoveryService);
+        var apiFactory = AgentTransportStrategy.Get(
+            exporterSettings,
+            productName: "debugger",
+            tcpTimeout: null,
+            AgentHttpHeaderNames.DefaultHeaders,
+            () => new DefaultJsonHeaderHelper(),
```

Comment:
As earlier, you will probably want/need new versions of both of these, and a reasonable timeout on the requests

Reply:
Headers changed here 360b832aff3dc02a2f871a46cb13bf52d25d75fe
Timeouts changed here 45eb61aa0314287522a7d1da6c1ab223df349342